admin-ui/stormer.py

```diff
@@ -0,0 +1,32 @@
+'''
+This class might seem kinda useless on it owns but has some rationale behind it.
+From the simplegui lib we have "listbox" (and I believe "Combo" as alt. for this) that give the functionality
+ needed to easily display a list. From listbox doc: 
+:param values:                     list of values to display. Can be any type including mixed types as long as they have __str__ method
+:type values:                      List[Any] or Tuple[Any]
+
+it offers one other usefull field namely "metadata of type 'any'". 
+
+So the problem is that I want to display the names of stormers but identify them by vunetID when looking them up in the
+ local cached data or pulling from db (cuz duplicates and such, duh!). Since listbox uses py's build in __str__ having
+  a list of tuples or something similiar is not functional since it will display the ID aswell. 
+I considered adding a sublist to the metadata param and retrieve the id based on 'index' of name in first list. This 
+would have been possible but I didn't like it, it also means that in case of duplicate names I have to extract the correct
+index from name list to get id, I believe this is possible with python but I was not sure.
+
+So my other idea was to just simply overwrite the __str__ method of my list of tuple objects, I figured that it probably would
+be possible but it seemed dirty. Hence this class just so I can have my own nice little str method.
+
+Might start using it to just store all data though in future, this would mean though that all cached user data ends up in the listbox...
+
+'''
+
+class Stormer():
+    def __init__(self, firstName, lastName, vunetId, balance):
+        self.firstName = firstName
```

Comment:
include "nickname" field as it's an attribute for a User in the db and is in the current knt system and i think we still want to keep it?

Reply:
Didnt know but cool! Will add.

Reply:
So I pulled an all-nighter for some other stuff, so my mind is exceptionally slow today and finally caught up. Are you sure there is a nickname in the current system? Because I think you might be confused by the middle part in dutch last names and assumed it's for nicknames. Basically, for my name "Nico de Gier" it would take it as First:"Nico", Insertion:"de," Last:"Gier". I don't know why they call it "insertion" instead of something like "middle" but could it be possible that you assumed that it is for a nickname?

Regardless of this, I think it highlights why we should store a name as first and last or just as first+last and not in the dutch manner since it's confusing!

Reply:
Well, yeah fair enough. Although I don't think very many people are using it as an insertion. However, in the interface of the KnT system in the StoKa, this "insertion" is between quotation marks. So that's why I think there is an actual "nickname" field